### DISCUSSION TITLE
Error in instalation due to Gandalf config?

### DISCUSSION THREAD

**Discussion by SantaTitular** [manujosephv/pytorch_tabular#343] [General]
**Describe the bug**
mportError: cannot import name 'GANDALFConfig' from 'pytorch_tabular.models'

I created a new conda environment and followed the installation steps but I get the error above due to C:\Users\tomas\anaconda3\envs\year3Tab_v2\lib\site-packages\pytorch_tabular\models\mixture_density\mdn.py:25: UserWarning: Wandb not installed. WandbLogger will not work.
  warnings.warn("Wandb not installed. WandbLogger will not work.")

See picture below. Could you help me? It works if I just dont import the GandalfConfig !

![Gandalf](https://github.com/manujosephv/pytorch_tabular/assets/68435474/fa53bde0-644f-42c5-9ad9-38d7a7ec01ca)

**Comment by manujosephv** [comment]
Gandalf is a new model which was added recently to the main branch. It's not part of the pypi package as of now. You'll either have to use a different model, or install directly from GitHub...
